manifests/k/kalilinux/kalilinux/1.13.1.0/kalilinux.kalilinux.locale.en-US.yaml

```
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: kalilinux.kalilinux
+PackageVersion: 1.13.1.0
+PackageLocale: en-US
+Publisher: Kali Linux
+Author: Kali Linux
+PackageName: Kali Linux
+PackageUrl: https://www.kali.org/tutorials/kali-on-the-windows-subsystem-for-linux/
```

```suggestion
PackageUrl: https://learn.microsoft.com/en-us/windows/wsl/install-manual#downloading-distributions
```
Yes, the InstallerUrl should be available on the PackageUrl or a few clicks away.